sinon/test/TestSinonSpy.py

```diff
@@ -857,6 +857,31 @@ def test240_neverCalledWithMatch_args(self):
         self.assertTrue(spy.neverCalledWithMatch("d", "e", "c")) #it's a combination
 
     @sinontest
+    def test250_firstCall_to_lastCall(self):
```

Comment:
I prefer to separate to two test functions:

`def test250_firstCall_to_lastCall_with_call(self):` -> spy1

`def test251_firstCall_to_lastCall_without_call(self):` -> spy2

Thanks!